config.drupal-xb-dev.yaml

```diff
@@ -17,3 +17,8 @@ webimage_extra_packages:
   - libxtst6
   - xauth
   - xvfb
+web_extra_exposed_ports:
+  - name: vite-dev-server
+    container_port: 5173
+    http_port: 5173
+    https_port: 5174
```

Comment:
We need to assign port numbers to both HTTP and HTTPS, even though we only use one of them.

Reply:
For posterity (e.g., future me 😉), this is documented here: [Extending and Customizing Environments - DDEV Docs](https://ddev.readthedocs.io/en/stable/users/extend/customization-extendibility/#exposing-extra-ports-via-ddev-router).

Reply:
My link from the description also leads to (one "step" before) this in the DDEV docs. 😊